### ISSUE TITLE
Ordered list-like lines should be escaped

### ISSUE THREAD

**Issue by marijnvdwerf** [thephpleague/html-to-markdown#73]
**Input**:
```html
<p>123456789) Foo and 1234567890) Bar!</p>
<p>1. Platz in 'Das große Backen'</p>
```

**Actual**:
```md
123456789) Foo and 1234567890) Bar!

1. Platz in 'Das große Backen'
```

**Expected**:
```md
123456789\) Foo and 1234567890) Bar!

1\. Platz in 'Das große Backen'
```

**Comment by andreskrey** [comment]
Just created #106. Check it out!

**Comment by colinodell** [comment]
Fixed via #106.
